update_requirements.sh

```diff
@@ -40,25 +40,29 @@ fi
 eval "$(pyenv init --path)"
 
 echo "update pip"
-pip install pip==21.3.1
-pip install pipenv==2021.11.23
+pip install pip==22.2.2
 
 if [ -d "./xx" ]; then
   echo "WARNING: using xx Pipfile ******"
-  export PIPENV_SYSTEM=1
   export PIPENV_PIPFILE=./xx/Pipfile
-fi
+  pip install pipenv==2021.11.23
+
+  echo "pip packages install..."
+  pipenv sync --dev
+  pipenv --clear
 
-if [ -z "$PIPENV_SYSTEM" ]; then
-  echo "PYTHONPATH=${PWD}" > .env
-  RUN="pipenv run"
-else
   RUN=""
+else
+  echo "PYTHONPATH=${PWD}" > .env
+  pip install poetry==1.2.1
+  poetry config virtualenvs.prefer-active-python true --local
```

Comment:
since we use pyenv to manage the python installation we should set this
https://python-poetry.org/docs/managing-environments/

Reply:
it also creates a `poetry.toml` file - added to `.gitignore` since it can include custom user configuration